<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>systemconfig</class>
 <widget class="QWidget" name="systemconfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>821</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="sd">
    <attribute name="title">
     <string>系统配置</string>
    </attribute>
    <widget class="QPushButton" name="set1_cancel">
     <property name="geometry">
      <rect>
       <x>649</x>
       <y>411</y>
       <width>85</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Config_OK">
     <property name="geometry">
      <rect>
       <x>501</x>
       <y>411</y>
       <width>85</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>251</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>充电枪数量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="gunnum">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>平台协议</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Pt_agreement">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>云快充</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>智绿</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>达克云</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>智融</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>蔚景云</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>充电模式</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cg_mode">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>IC卡</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>密码</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>后台</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>辅助电源</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Auxiliarypower">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>启用自选</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>停用</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>默认12V</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>默认24V</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>单车充电模式</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Singlevehmode">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>启用</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>停用</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>充电桩类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Piletype">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>双枪分配</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>双枪双充</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>监控参数</string>
    </attribute>
    <widget class="QPushButton" name="Monitored_cancel">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>410</y>
       <width>85</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>276</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>充电机类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Cgtype">
          <property name="styleSheet">
           <string notr="true">border : 2px solid black</string>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>双枪双充</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>双枪分配</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_26">
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>输出电压</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Outvol"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_27">
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>输出电流</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Outcur"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>绝缘告警值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="InsulationAlrmValue"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>电池最高温度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Maxbatterytemp"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>电池单体最高电压</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Maxsinbatteryvol"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>充电枪最高温度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Maxguntemp"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>充电枪最高温度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="InsulationDete">
          <property name="styleSheet">
           <string notr="true">border : 2px solid black</string>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>启用</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>停用</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>331</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>车辆连接检测</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="VehconnectDete">
          <property name="styleSheet">
           <string notr="true">border : 2px solid black</string>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>启用</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>停用</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>充电枪温度检测</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="GuntempDete">
          <property name="styleSheet">
           <string notr="true">border : 2px solid black</string>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>启用</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>停用</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>电子锁故障检测</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ElecLockFaultDete">
          <property name="styleSheet">
           <string notr="true">border : 2px solid black</string>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>启用</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>停用</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>绝缘模块通讯规约</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="InsulationModuleProtocol">
          <property name="styleSheet">
           <string notr="true">border : 2px solid black</string>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>单路</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>双路</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_21">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>充电模块通讯规约</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ChargingModuleProtocol">
          <property name="styleSheet">
           <string notr="true">border : 2px solid black</string>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>英飞源</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_22">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>电表通讯规约</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ElecMeterProtocol">
          <property name="styleSheet">
           <string notr="true">border : 2px solid black</string>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>千双</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>双路</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>千单</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>单路</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_23">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>风扇延迟</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="FanDelay"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="Monitored_OK">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>410</y>
       <width>85</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>桩基本数据</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>桩编码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Pilenum"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>充电员密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Charger_password"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>管理员密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Admin_password"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>371</width>
       <height>35</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>清除交易记录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearrecords">
        <property name="text">
         <string>清除</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="Piledata_cancel">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>390</y>
       <width>85</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>费率</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
